Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3
+FROM alpine@sha256:2171658620155679240babee0a7714f6509fae66898db422ad803b951257db78
```

Comment:
Do you specifically want to lock to this version? The concern would be that we might miss security patches that make it into alpine v3

Reply:
Pinned it back to version v3